lib/ansible/modules/user.py

```diff
@@ -204,13 +204,13 @@
             - Lock the password (C(usermod -L), C(usermod -U), C(pw lock)).
             - Implementation differs by platform. This option does not always mean the user cannot login using other methods.
             - This option does not disable the user, only lock the password.
-            - This must be set to V(False) in order to unlock a currently locked password. The absence of this parameter will not unlock a password.
+            - This must be set to V(false) in order to unlock a currently locked password. The absence of this parameter will not unlock a password.
             - Currently supported on Linux, FreeBSD, DragonFlyBSD, NetBSD, OpenBSD.
         type: bool
         version_added: "2.6"
     local:
         description:
-            - Forces the use of "local" command alternatives on platforms that implement it.
+            - Forces the use of I(local) command alternatives on platforms that implement it.
```

Comment:
We actually should swap out `local` for `libuser` and deprecate the `local` alias, see https://github.com/ansible/ansible/issues/76376. I'm not sure I see the benefit of using italic.

Reply:
It is a bit more complex as sometimes withi `local` we edit `/etc/passwd` directly.

Also, I found out that  'module option deprecation' was not an allowed feature in modules docs,
but the following enables it #82593, though we don't have a 'common way' of deprecating aliases.

Reply:
Hi @s-hertel there was nothing very objective about the italic there. As it seems we do not have any specific recommendation towards that so it is based on subjectivity. I am reverting that change, based on that fact. Thanks for commenting on it.